_drafts/2017-08-20-getting-data.md

```diff
@@ -43,29 +41,27 @@ def make_requestion(game_id):
 ```
 
 ## Getting the game description
-Using the inspect tool in e.g. Google Chrome we can see that the game description has been given a name space.  Beautiful Soup allow us to give a CSS selector.
+Using the inspect tool in e.g. Google Chrome we can see that the HTML tags that contain the game description has its `name` attribute set to `Description`. Beautiful Soup allow us to extract the text of that tag using a CSS selector.
 
 ```python
 def get_desctiption(soup):
     desc_tag = soup.select('[name="Description"]')
-    if len(desc_tag)>0:
+    if len(desc_tag) > 0:
         desc = str(desc_tag[0])[15:-22]
```

Comment:
Is this really necessary? Can't you use `get_text().strip()` or similar as you do below?